packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -125,16 +134,37 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, ITooltipHostSt
     }
 
     this._toggleTooltip(true);
+    this._clearDismissTimer();
+  }
+
+  // Hide Tooltip
+  private _onTooltipMouseLeave = (ev: any): void => {
+    if (this.props.closeDelay) {
+      this._clearDismissTimer();
+
+      this._closingTimer = this._async.setTimeout(() => {
+        this._toggleTooltip(false);
+      }, this.props.closeDelay);
+    } else {
+      this._toggleTooltip(false);
+    }
+  }
+
+  private _clearDismissTimer = (): void => {
+    window.clearTimeout(this._closingTimer);
```

Comment:
(thought I had commented on this before, somehow got lost..)

you should use the `_async` utility.